specs/packages/sdk.md

```diff
@@ -0,0 +1,45 @@
+# Sdk
+
+Sdk is a part of shared package. It's main purpose is to wrap shared logic from the namada crate, so it can be used in the context of the browser.
+
+Modules:
+
+- [masp](#masp)
+- [tx](#tx)
+- [wallet](#wallet)
+- [sdk](#sdk)
+
+## masp
+
+Contains implementation of the `ShieldedUtils` trait. It is responsible for:
+
+- loading and saving spend, convert and output params into and from the browser storage.
+- serializing and deserializing params for read/write
+- creating LocalTxProver needed for creating MASP proof
+
+## tx
+
+Encapsulates functionality for deserialization and mapping of byte arrays into Rust structures.
+Contains:
+
+- Definitions of structs representing transaction messages. Message is a BorshSerialized JsValue that represents all the data needed to create a transaction.
+- Mapping functions which deserialize transaction messages and map those for proper transaction arguments
+
+## wallet
+
+Contains the logic for:
+
+- encoding/decoding wallet store
+- adding keys/spending kest
+
+### Why we are not using key generation functions from the Namada Shared?
+
+At the moment Anoma extension creates keys based on mnemonic, while Namada uses OsRng as a source of randomness. Because of that we have to introduce functions that allow to add keys generated(derived) inside JS codebase into wallet storage. This will most likely change in future after Namada introduces mnemonic support.
```

Comment:
I was going to suggest that we link here to our current "Key Managment" docs, but they are not particularly informative: https://github.com/anoma/namada-interface/blob/main/specs/browser-extension/key-management.md - so, I think this is fine. I would maybe slightly re-word it to something like the following (if I understand this part correctly):

```
Currently, the Anoma extension derives keys based on a mnemonic, while Namada derives keys based on a source of randomness (`OsRng`). As such, we have introduced functions that allow us to add keys derived within the extension into Sdk's wallet storage. This will most likely change in future after Namada introduces support for mnemonics into the Sdk.
```

Something like that?